Automattic/wp-calypso
# Signup: Streamlined TOS block

This PR tries to simplify and streamline a few things on the account screen on sign up. It changes the button labels to be consistent for all three authentication options, and uses simpler language to describe the terms. 

See below for changes in detail.

Before | After
------ | ------
![screencapture-wordpress-start-user-2019-10-07-10_34_05](https://user-images.githubusercontent.com/426518/66294798-830e6880-e8f2-11e9-930c-0e7a58dd8719.png) | ![screencapture-wordpress-start-user-2019-10-07-10_34_05](https://user-images.githubusercontent.com/205419/71246600-4c53c600-2317-11ea-8f4f-90f2d15ed6c6.png)

### Action labels

Before:  
> Create your account / Continue with Google / Continue with Apple

After:  
> Continue with email / Continue with Google / Continue with Apple


The current labels make the different options seem more different then they are, and can suggest that only the first option creates an account. 

Using a consistent pattern for the label should make it clearer that all three options are equivalent, and highlight the one thing they do differ on: authentication relying either on your email, on Google, or on Apple. 

### Terms

Before:  
> By creating an account, you agree to our Terms of Service.
>
> [...]
>
> If you continue with Google or Apple and don't already have a WordPress.com account, you are creating an account and you agree to our Terms of Service.

After:  
> By continuing, you are creating an account on WordPress.com, and you agree to our [Terms of Service].

Changing the button label also makes it easier to explain the terms for all three options at once, and removes the need for the lengthy terms section below the social buttons.

It looks like the clarification for Google and Apple may have originated from the log in screen – where is there more room for confusion about creating an account. I don't think this is needed on the sign up screen.

### Heading for social options

Before:
> Or create an account using:

After:
> Or connect your existing profile to get started faster.

This brings back some context for the social options.

### Test plan

1. Confirm the signup form looks like on "After" screenshot above
1. Confirm that login form is still using the old copy and makes sense in general:

<img width="471" alt="Zrzut ekranu 2019-12-20 o 10 54 58" src="https://user-images.githubusercontent.com/205419/71246823-c421f080-2317-11ea-9fe3-1252f47beb07.png">

1. Go through other flows using the components we're updating here, specifically:
* CrowdSignal login (`http://calypso.localhost:3000/log-in?client_id=978&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26`)
* Another CrowdSignal (`/start/crowdsignal/oauth2-name/en`)
* Main signup flow (`/start/main/user/en`)
* Design-first signup flow (`/start/design-first/template-first-themes/en`)
* E-commerce signup flow (`/start/ecommerce/user/en`)
* Reader flow (`/start/reader/reader-landing/en`)
* Account flow (`/start/account/user/en`)
* WPCC (`/start/wpcc/oauth2-user/en`)
* Import (`http://calypso.localhost:3000/start/import/user/en`)

### Links

p4H3ND-SV-p2
paObgF-E4-p2
